src/pools/abstract-pool.ts

```diff
@@ -437,6 +448,32 @@ export abstract class AbstractPool<
     }
   }
 
+  /**
+   *
+   * @param worker
```

Comment:
Please run the linter and see where you need to add some docs
We try to serve a well documented codebase

Reply:
It's a draft PR tagged RFC not merge ready.